### ISSUE TITLE
Bad doc link

### ISSUE THREAD

**Issue by brycenesbitt** [Rediker-Software/doac#13]
The link to "views" from https://github.com/Rediker-Software/doac/blob/master/docs/index.md is bad.  Without good docs, people won't adopt the App.

**Comment by Fingel** [comment]
Yup, this repo sounds like what I'm looking for, but the documentation is incomplete and I have no idea what it actually does.
